examples/guides/messaging/messaging-with-sqs/messaging-with-sqs.md

```diff
@@ -0,0 +1,298 @@
+        secretKey: config:getAsString("SECRET_ACCESS_KEY"),
+        region: config:getAsString("REGION"),
+        accountNumber: config:getAsString("ACCOUNT_NUMBER")
+    };
+
+    // Amazon SQS client
+    amazonsqs:Client sqsClient = new(configuration);
+    string receivedReceiptHandler = "";
+
+    // Receive a message from the queue
+    map<string> attributes = {};
+    // MaxNumberOfMessages, the maximum number of messages that can be received per request
+    attributes["MaxNumberOfMessages"] = "10";
+    // VisibilityTimeout, time allowed to delete after received, in seconds
+    attributes["VisibilityTimeout"] = "2";
+    // WaitTimeSeconds, waits for this time (in seconds) till messages are collected before received
+    attributes["WaitTimeSeconds"] = "1";
+
+    while(true) {
+
+        // Wait for 5 seconds
+        runtime:sleep(5000);
+        // Receive messages from the queue
+        amazonsqs:InboundMessage[]|error response = sqsClient->receiveMessage(queueResourcePath, attributes);
+
+        // When the response is not an error
+        if (response is amazonsqs:InboundMessage[]) {
+
+            // When there are messages available in the queue
+            if (response.length() > 0) {
+                log:printInfo("************** Received fire alerts! ******************");
+                int deleteMssageCount = response.length();
+                log:printInfo("Going to delete " + deleteMssageCount.toString() + " messages from queue.");
+                
+                // Iterate on each message
+                foreach var eachResponse in response {
+                    
+                    // Keep receipt handle for deleting the message from the queue
+                    receivedReceiptHandler = eachResponse.receiptHandle;
+
+                    // Delete the received the messages from the queue
+                    boolean|error deleteResponse = sqsClient->deleteMessage(queueResourcePath, receivedReceiptHandler);
+                    
+                    // When the response from the delete operation is valid
+                    if (deleteResponse is boolean && deleteResponse) {
+                        if (deleteResponse) {
+                            log:printInfo("Deleted the fire alert \"" + eachResponse.body + "\" from the queue.");
+                        }
+                    } else {
+                        log:printError("Error occurred while deleting the message.");
+                    }
+                }
+            } else {
+                log:printInfo("Queue is empty. No messages to be deleted.");
+            }
+
+        } else {
+            log:printError("Error occurred while receiving the message.");
+        }
+    }
+}
+```
+
+In the following code, the main method would implement the workers related to creating a queue, sending a message to the queue consuming and receiving / deleting messages form the queue.
+
+**main.bal**
```

Comment:
Shall we add this code segment using INCLUDE_CODE tag?
Please refer https://github.com/wso2/ballerina-integrator/tree/master/www#a-include-a-code-file

Reply:
Added INCLUDE_CODE tag and moved the code to correct guide location.